pkg/build/builder/common.go

```diff
@@ -20,7 +22,14 @@ import (
 // client facing libraries should not be using glog
 var glog = utilglog.ToFile(os.Stderr, 2)
 
-const OriginalSourceURLAnnotationKey = "openshift.io/original-source-url"
+const (
+	OriginalSourceURLAnnotationKey = "openshift.io/original-source-url"
```

Comment:
@bparees so this is used for the case where we have to use a proxy because the password is too long to be used directly with git. True, it was mainly for the use case of the git server because we use service account tokens... but it could be used in other cases where you have a long password/token. It was only a temporary solution until git supported long passwords in centos/rhel. With 7.3 it's no longer necessary.